src/twisted/names/server.py

```diff
@@ -224,11 +224,29 @@ def _responseFromMessage(
             authority = []
         if additional is None:
             additional = []
+
+        # This authoritativeAnswer logic might not be entirely correct
```

Comment:
would this perhaps be better expressed by filing (and linking to) an issue and the RFC rather than pasting the entire body of the spec in here?